### PR TITLE
Add `RawTask` executing docs ad verbatim

### PR DESCRIPTION
**Pull request by rmitsch**
## Description

Add `RawTask` executing docs ad verbatim, i. e. the surrounding prompt is minimal and doesn't give any instructions except for "read and reply". This is useful for use cases where each doc has its individual prompt or for question answering.

### Corresponding documentation PR

https://github.com/explosion/spaCy/pull/13180

### Types of change

New feature.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran all tests in `tests` and `usage_examples/tests`, and all new and existing tests passed. This includes
  - all external tests (i. e. `pytest` ran with `--external`)
  - all tests requiring a GPU (i. e. `pytest` ran with `--gpu`)
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by rmitsch** [comment]
Not sure about the name, open to suggestions.

File: spacy_llm/tasks/raw/registry.py
Comment (svlandeg) [inline spacy_llm/tasks/raw/registry.py:13]:
```suggestion
def make_raw_parser() -> TaskResponseParser[RawTask]:
```

File: spacy_llm/tasks/raw/registry.py
Comment (svlandeg) [inline spacy_llm/tasks/raw/registry.py:25]: Considering that this is a field on the `Doc` level, maybe something slightly more specific, like `llm_response` or `llm_reply` or such?

File: spacy_llm/tasks/raw/registry.py
Comment (svlandeg) [inline spacy_llm/tasks/raw/registry.py:42]:
```suggestion
```

File: spacy_llm/tasks/raw/task.py
Comment (svlandeg) [inline spacy_llm/tasks/raw/task.py:36]:
```suggestion
```

File: spacy_llm/tasks/templates/raw.v1.jinja
Comment (svlandeg) [inline spacy_llm/tasks/templates/raw.v1.jinja:19]: If there are no examples, wouldn't it make more sense to just push the `{{text}}` out without any further instructions?

**Comment by svlandeg** [comment]
> Not sure about the name, open to suggestions.

Naming, my biggest forte!

`VerbatimTask`? 😁

File: spacy_llm/tasks/templates/raw.v1.jinja
Comment (svlandeg) [reply spacy_llm/tasks/templates/raw.v1.jinja:19]: It would also be more "true" to the promise of just sending the doc's content...

File: spacy_llm/tasks/raw/registry.py
Comment (rmitsch) [reply spacy_llm/tasks/raw/registry.py:39]: Agreed, but yeah - it still can make sense.

File: spacy_llm/tasks/templates/raw.v1.jinja
Comment (rmitsch) [reply spacy_llm/tasks/templates/raw.v1.jinja]: Removed everything except for "Text:" and "Reply:". Do you thing we should toss these too?

**Comment by rmitsch** [comment]
> VerbatimTask? 😁

Hm, not that much of a fan (`RawTask` isn't any better). Something with "direct", "as-is"...I'll think about it a bit more. Feel free to leave more suggestions :grin:

**Review by svlandeg — APPROVED** [review]
Good to merge after resolving the merge conflicts!

**Pull request merged**
